docs/resources/tunnel_route.md

```diff
@@ -26,15 +26,15 @@ resource "cloudflare_tunnel_route" "example" {
 }
 
 # Tunnel with tunnel route
-resource "cloudflare_argo_tunnel" "tunnel" {
```

Comment:
this will need to be changed at https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/examples/resources/cloudflare_tunnel_route/resource.tf#L11